Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1752,3 +1752,19 @@ def remove_nulls_from_dictionary(dict):
     for key in list_of_keys:
         if dict[key] in ('', None, [], {}, ()):
             del dict[key]
+
+
+def get_demisto_version():
+    """
+        Returns Demisto version and build number
+
+        :return: Demisto version object
+        :rtype: ``dict``
+    """
+    if hasattr(demisto, 'demistoVersion'):
+        return demisto.demistoVersion()
+    else:
+        return {
+            'version': '0.0.0',
```

Comment:
isn't it better to return an error msg if you cannot fetch the version, rather than returning magic number that looks like  a valid version?

Reply:
it would require wrapping it in `try-except` block if used in an integration, for example
the idea was to avoid it and get the version, and then be able to compare between versions

Reply:
I dont see how warping in an `if` statement and comparing it to "0.0.0" is better than `try-except`.
and if you are just checking if the answer equals the version you expect (say, `if res['version'] == '5.0.0':`),
than you may never know that it is not that you didn't got the right version but you didn't get a version at all.